ens/ens.py

```diff
@@ -87,6 +87,9 @@ class ENS(BaseENS):
     like: ``"0x314159265dD8dbb310642f98f50C066173C1259b"``
     """
 
+    # mypy types
+    w3: "Web3"
```

Comment:
We do since `BaseENS` defines `Union["ENS", "AsyncENS"]`. I can either be here or it can be in the `__init__()` method when we assign it `self.w3: "Web3" = init_web3(...)`. It seemed consistent with the other `Module` to set it this way but I could go either way. Any preference?

Reply:
Got it. No preference, I was mostly just wondering!